hil/migrations/versions/76529f0f9e50_add_uuid_and_status_to_networkingaction.py

```diff
@@ -0,0 +1,42 @@
+"""add uuid and status to NetworkingAction
+
+Revision ID: 76529f0f9e50
+Revises: 9089fa811a2b
+Create Date: 2018-01-07 15:24:09.545021
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '76529f0f9e50'
+down_revision = '9089fa811a2b'
+branch_labels = None
+
+# pylint: disable=missing-docstring
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('networking_action', sa.Column('status', sa.String(),
+                  nullable=True))
+    op.add_column('networking_action', sa.Column('uuid', sa.String(),
+                  nullable=True))
+    op.create_index(op.f('ix_networking_action_uuid'), 'networking_action',
+                    ['uuid'], unique=False)
+    op.execute("UPDATE networking_action SET status = 'PENDING', \
+        uuid ='no-uuid'")
+    op.alter_column('networking_action', 'status', nullable=False)
+    op.alter_column('networking_action', 'uuid', nullable=False)
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index(op.f('ix_networking_action_uuid'),
+                  table_name='networking_action')
+    op.drop_column('networking_action', 'uuid')
+    op.drop_column('networking_action', 'status')
```

Comment:
Observation: this will do strange things if there are actions in the table whose status is not `'PENDING'`. I'm Ok just reporting an error in that case, but we shouldn't just silently corrupt the information.

Reply:
alright, so I am just deleting those now, no point in keeping those if we can't query them.